sut/modules/unish/woot/drush.services.yml

```diff
@@ -17,3 +17,7 @@ services:
     class: Drupal\woot\Commands\AnnotatedGreetCommand
     tags:
       -  { name: console.command }
+  woot.command_alterer:
```

Comment:
I'm unsure if you want this change, but my thought process was that this example should be complete. 🤔